src/Styleguide/Elements/Button.tsx

```diff
@@ -76,10 +75,12 @@ export const Button = styled(
                 border-color: ${colors.black100};
                 color: ${colors.white100};
 
-                &:hover {
-                  background-color: ${colors.purple100};
-                  border-color: ${colors.purple100};
-                  color: ${colors.white100};
+                @media ${themeProps.mediaQueries.hover} {
```

Comment:
…and here’s one using just the media query.

Reply:
Oh didn’t know about that one! I see that it’s still referring to just breakpoints as well. I can follow up with this later 👍